src/main/java/teammates/ui/controller/InstructorCourseEnrollResultPageData.java

```diff
@@ -8,27 +9,77 @@
 import teammates.common.datatransfer.StudentAttributes.UpdateStatus;
 import teammates.common.util.Const;
 import teammates.common.util.Utils;
+import teammates.ui.template.EnrollResultPanel;
 
 /**
  * PageData: page data for the 'Result' page after enrollment for a course
  */
 public class InstructorCourseEnrollResultPageData extends PageData {
+    protected static final Logger log = Utils.getLogger();
+    
+    private String courseId;
+    private List<StudentAttributes>[] students;
+    private boolean hasSection;
+    private String enrollStudents;
+    private List<EnrollResultPanel> enrollResultPanelList;
     
-    public InstructorCourseEnrollResultPageData(AccountAttributes account) {
+    public InstructorCourseEnrollResultPageData(AccountAttributes account, String courseId, 
+                                                List<StudentAttributes>[] students, boolean hasSection, 
+                                                String enrollStudents) {
         super(account);
+        this.courseId = courseId;
+        this.students = students;
+        this.hasSection = hasSection;
+        this.enrollStudents = enrollStudents;
+        enrollResultPanelList = new ArrayList<EnrollResultPanel>();
+        
+        for (int i = 0; i < UpdateStatus.STATUS_COUNT; i++) {
+            String panelClass = "";
+            
+            switch (UpdateStatus.enumRepresentation(i)) {
+            case ERROR :
+                panelClass = "panel-danger";
+                break;
+            case NEW :
+                panelClass = "panel-primary";
+                break;
+            case MODIFIED :
+                panelClass = "panel-warning";
+                break;
+            case UNMODIFIED :
+                panelClass = "panel-info";
+                break;
+            case NOT_IN_ENROLL_LIST :
+                panelClass = "panel-default";
+                break;
+            default :
+                panelClass = "panel-danger";
+                break;
+            }
```

Comment:
For the sake of consistency, we may want to also check for an `UNKNOWN` case and set the `default` as a call to the logger.

Also, in our Java style guide, we give another level of indentation to the case blocks in a switch-case.